frontend/src/features/mapSideBar/SidebarContents/BuildingForm.tsx

```diff
@@ -339,11 +333,19 @@ const BuidingForm: React.FC<IParentBuildingForm> = ({
   const initialValues = {
     activeStep: 0,
     activeTab: 0,
-    data: { ...defaultBuildingValues, ...rest.initialValues },
+    data: {
+      ...defaultBuildingValues,
+      ...rest.initialValues,
+      evaluations: getMergedFinancials(
```

Comment:
Same thing, setting up the extra empty evaluations/fiscals (whilst keeping any existing ones).